src/customer.ts

```diff
@@ -97,9 +98,13 @@ import {
     PreReportHook,
     MutateResourceOperation,
     ReportStreamOptions,
+    CreateCustomerOptions, CreateCustomerFlowSettings,
 } from './types'
 
 export interface CustomerInstance {
+    /* Base Customer properties for easy access */
+    cid: string,
```

Comment:
We should probably use `readonly` here, so people are aware that this shouldn't be changed
```ts
readonly cid: string,
```

Reply:
Changed it to readonly, goog catch, thank you.